Godeps/_workspace/src/bazil.org/fuse/syscallx/syscallx_std.go

```diff
@@ -3,24 +3,24 @@
 package syscallx
 
 // This file just contains wrappers for platforms that already have
-// the right stuff in stdlib.
+// the right stuff in golang.org/x/sys/unix.
 
 import (
-	"syscall"
+	"golang.org/x/sys/unix"
```

Comment:
@tv42  I was confused that `godep` didn't vendor `golang.org/x/sys/unix` and didn't rewrite the import path here to `prefix/_workspace/src/...`

My (way too nebulous) question for you was if the `bazil.org/fuse/fs -> bazil.org/fuse/syscallx -> golang.org/x/sys/unix` import chain is somehow conditional. I could have checked this myself this morning - `syscallx` is only included in the __tests__ of `bazil.org/fuse/fs` so godep is correct (it doesn't respect test deps at all) with not including it. 

My brain was only faking higher states of awareness, I guess. Sorry for the noise.

@jbenet we can move forward.